portfolio_ahardman.html

```diff
@@ -0,0 +1,71 @@
+<!doctype html>
+<html lang='en'>
+<head>
+	<title>portfolio_ahardman</title>
+	<link href='normalize.css' rel='stylesheet' type='text/css'>
+	<link href='portfolio_ahardman_stylesheet.css' rel='stylesheet' type='text/css'>
+</head>
+<body>
+	<div class='navbar'>
+		<div class='left container'>
+		</div>
+		<div class='link container'>
+			<a href='#' alt='test_link_1'>Home</a>
```

Comment:
I didn't know hash did this and this works great for this page, but keep in mind that if you plan to use this nav for other pages this won't link to the home page. (i think lol)